scripts/test-liquiditySwap-swapAndDeposit-swapAll-WETH-DAI.js

```diff
@@ -26,42 +23,59 @@ async function main() {
   // Build a swap on ParaSwap
   const amount_to_swap = ethers.utils.parseEther('10.01');
   const paraswap = new ParaSwap(parseInt(FORK_NETWORK_ID), PARASWAP_API);
-  const priceRoute = await paraswap.getRate('WETH', 'DAI', amount_to_swap.toString(), 'SELL', { referrer: 'aave', excludeDEXS: 'Balancer', excludeMPDEXS: 'Balancer' });
+  const priceRoute = await paraswap.getRate(
+    'WETH',
+    'DAI',
+    amount_to_swap.toString(),
+    'SELL',
+    { referrer: 'aave', excludeDEXS: 'Balancer', excludeMPDEXS: 'Balancer' }
```

Comment:
@ColonelJ , what#s the reasoning behind disabling balancer?

Reply:
Our implementation of Balancer encodes the exact input amounts for each of the pools, so if you try to use it with swap all balance it won't work.